### ISSUE TITLE
Add option for maximum area size

### ISSUE THREAD

**Issue by heapoutofspace** [SimpleServer/SimpleServer#167]
Of course only for areas defined with `/myarea`

**Comment by sk9c00** [comment]
I gotta agree... and maybe more "flags" for area protection...

**Comment by heapoutofspace** [comment]
What exactly do you mean by that?

**Comment by sk9c00** [comment]
Currently as I see it, there is no options for redefining the maximum size of an area, i.e. everyone is limited to a "quota" of area even admins. I know there are "ignore area" but is there a "max area/volume"? And also maybe an option to the maximum allowed separate areas a person is allowed.

By "flags" I mean a way to edit your settings of your personal area. Like add a friend to build in your area, or allow a person to create chests in your area, and Maybe also add an option where you can control who can access your chest.
Maybe even a reload area function... <== Ok maybe this is asking for too much...

**Comment by sk9c00** [comment]
Oh is it possible to protect areas from tnt explosion, creeper explosion, etc... a bit like world guard - Bukkit Plugin.

Then you would have to add in a option to enable and disable different features won't you...

**Comment by heapoutofspace** [comment]
There's no way we could implement that. For disabling creeper explosions I use a little [mod](https://github.com/sadimusi/CreeperNerf) I made some time ago, there are probably others available with different features.

**Comment by sk9c00** [comment]
Is there an anti-block damage protocol, as in disable all 'environmental' damages in an area like filter off all events in an area, or maybe have a save area and area reload option?

**Comment by heapoutofspace** [comment]
No, none of this is possible. This is a network wrapper, we can't influence the inner working of the server.

**Comment by sk9c00** [comment]
But how can you prevent people to place blocks in an area?
Original... I mean...

**Comment by lucifer6642** [comment]
I thought AreaMaxX and AreaMaxZ in the config.xml file affected how big the /myarea stuff can be?
